submissions/ToyinAyorinde/Html/LandingPage/landingpage.css

```diff
@@ -0,0 +1,236 @@
+body{
```

Comment:
Please work on this file, I can see improper indentation in some places and unnecessary spaces before closing curly braces.